tests/modulefunctions_test.py

```diff
@@ -105,3 +106,8 @@ def testUnstubFunctionOnModuleWhichIsActuallyAMethod_issue_53(self):
         assert random.randint(1, 10) == "mocked"
         unstub(random)
         assert random.randint(1, 10) != "mocked"
+
+    def testStubsNotStrict(self):
```

Comment:
```suggestion
    def testAddFakeMethodInNotStrictMode(self):
```

The before untested functionality is to **add** new methods to an object.